### PR TITLE
Update for `stream_get_latency`

### PR DESCRIPTION
**Pull request by padenot**
This is necessary after https://github.com/kinetiknz/cubeb/pull/583

---
This change is [Reviewable](https://reviewable.io/reviews/djg/cubeb-rs/53)

### CODE REVIEW

**Comment by padenot** [comment]
I'll update the `libcubeb` sha later, I don't know if my commit in the other repo is correct, and the cubeb CI is broken.

File: cubeb-backend/src/capi.rs [hunk 1]
```diff
@@ -49,6 +49,7 @@ macro_rules! capi_new(
                 Some($crate::capi::capi_stream_reset_default_device::<$stm>),
             stream_get_position: Some($crate::capi::capi_stream_get_position::<$stm>),
             stream_get_latency: Some($crate::capi::capi_stream_get_latency::<$stm>),
+            stream_get_input_latency: Some($crate::capi::capi_stream_get_input_latency::<$stm>),
```
Comment (ChunMinChang) [inline cubeb-backend/src/capi.rs:52]: With `stream_get_input_latency` introduced, we will have two similar function names
- `stream_get_input_latency`
- `stream_get_latency`

How about renaming `stream_get_latency` to `stream_get_output_latency`? It definitely looks more understandable to me.

Comment (padenot) [reply cubeb-backend/src/capi.rs:52]: It's unnecessary ABI breakage, but yeah.

**Comment by padenot** [comment]
@kinetiknz, can you publish `0.7.0` with this, so I can update `cubeb-coreaudio-rs` and do a cubeb uplift ? Thanks!

**Comment by kinetiknz** [comment]
> @kinetiknz, can you publish `0.7.0` with this, so I can update `cubeb-coreaudio-rs` and do a cubeb uplift ? Thanks!

Done + I sent you crate ownership invite

**Pull request merged**